ui/app/components/http-requests-bar-chart.js

```diff
@@ -28,11 +31,11 @@ import { task, waitForEvent } from 'ember-concurrency';
 
 const HEIGHT = 240;
 
+const HOVER_PADDING = 12;
+
 export default Component.extend({
   classNames: ['http-requests-bar-chart-container'],
   counters: null,
-
-  /* eslint-disable ember/avoid-leaking-state-in-ember-objects */
```

Comment:
Guessing linting still complains about this - you could get around it by returning the object from a computed property like this: `computed(function(){ return <object here> })`

Reply:
linting isn't complaining about this :o

Reply:
Huh odd - was it at some point? Seems weird that you had the disable directive there before 🤔

Reply:
There's this in circle: 
```
/src/ui/app/components/http-requests-bar-chart.js
  36:3  error  Only string, number, symbol, boolean, null, undefined, and function are allowed as default properties  ember/avoid-leaking-state-in-ember-objects

✖ 1 problem (1 error, 0 warnings)

error Command failed with exit code 1.
```

Maybe something changed in your editor setup?

Reply:
Another option to avoid this linting objection is `Object.freeze`, like [this example](https://discuss.emberjs.com/t/working-around-avoid-leaking-state-in-ember-objects-eslint-rule/14864/13).

Reply:
Oh that's nice - I hadn't seen that @backspace

Reply:
oh, that's so handy! thank you for the example. i added it in!

Reply:
this is the best PR ever! 🙌 everyone!